lib/actions/deploy.coffee

```diff
@@ -196,6 +209,11 @@ module.exports =
 		appName = appName_raw || appName || options.application
 		delete options.application
 
+		options.convertEol = options['convert-eol'] || false
+		delete options['convert-eol']
+		if options.convertEol and not options.build
+		 exitWithExpectedError('The --eol-conversion flag is only valid with --build.')
```

Comment:
What I had been doing in the recent past was use `throw ExpectedError` instead of `exitWithExpectedError`. In this case, I guess, it might be a matter of calling `done()`:
```
    return done(new ExpectedError('The --eol-conversion flag is only valid with --build.'))
```